assets/app/view/game_card.rb

```diff
@@ -148,55 +147,84 @@ def render_link(href, click, text)
     def render_body
       props = {
         style: {
-          'margin-top': '0.5rem',
           'line-height': '1.2rem',
+          padding: '0.3rem 0.5rem',
           'word-break': 'break-all',
         },
       }
 
       p_elm = players.map.with_index do |player, index|
-        elm = h(
-          acting?(player) ? :u : :span,
-          { style: { 'margin-right': '0.5rem' } },
-          player['name'] + (index != (players.size - 1) ? ',' : ''),
-        )
-
+        short_name = player['name'].length > 19 ? player['name'][0...18] + '…' : player['name']
         if owner? && new? && player['id'] != @user['id']
           button_props = {
             on: { click: -> { kick(@gdata, player) } },
+            attrs: {
+              title: "kick #{player['name']}",
+            },
             style: {
-              'margin-left': '0.5rem',
+              padding: '0.1rem 0.3rem',
+              margin: '0 0.3rem 0.1rem 0.3rem',
             },
           }
-          elm = h('button.button', button_props, [elm])
-        end
 
+          elm = h('button.button', button_props, short_name)
+        else
+          player_props = { attrs: { title: player['name'].to_s } }
+
+          elm = h(:span, [
+            h(acting?(player) ? :em : :span, player_props, short_name),
+            index == (players.size - 1) || owner? && new? ? '' : ', ',
+          ])
+        end
         elm
       end
 
       children = [
-        h(:div, [h(:b, 'Id: '), @gdata['id'].to_s]),
-        h(:div, [h(:b, 'Description: '), @gdata['description']]),
-        h(:div, [h(:b, 'Players: '), *p_elm]),
+        h(:div, [h(:strong, 'Id: '), @gdata['id'].to_s]),
+        h(:div, [h(:strong, 'Description: '), @gdata['description']]),
+        h(:div, [h(:strong, 'Players: '), *p_elm]),
       ]
 
       if new?
-        children << h(:div, [h(:b, 'Max Players: '), @gdata['max_players']])
-        children << h(:div, [h(:b, 'Created: '), @gdata['created_at']])
+        children << h(:div, { style: { display: 'inline' } }, [h(:strong, 'Max Players: '), @gdata['max_players']])
```

Comment:
don't you have css class inline?

Reply:
O tempora! O mores! toby pushes CSS ;)
You did create it for inline-block (yay for blame). Will change.